src/db/transactions/updateTx.ts

```diff
@@ -21,7 +22,9 @@ export const updateTx = async ({
   txData,
 }: UpdateTxParams) => {
   const prisma = getPrismaWithPostgresTx(pgtx);
-
+  logger.server.debug(
+    `Updating transaction with queueId ${queueId}, status ${status}}`,
```

Comment:
For these, we should follow the format we're using elsewhere with the `[Transaction] [${queuId}] Message`

Reply:
these are debug logs.,I will remove them ventually. But yes, will follow this convention if I use `.info`